### ISSUE TITLE
install problems may require jailbreak

### ISSUE THREAD

**Issue by GoogleCodeExporter** [StonedRightMeow17/itransmission#75]
```
What steps will reproduce the problem?
1. installing the app itransmission like normal apps
2.
3.

What is the expected output? What do you see instead?

should install like normal apps instead getting invalid app

What version of the product are you using? On what operating system?

ipad 2 wifi only with ios 4.3.5 pre installed

Please provide any additional information below.

do i need to jailbreak in order to install? if so i am stuck i am unable to 
jailbreak because apple no longer uses shsh blobs to sign updates and there is 
no jailbreak for 5.x yet.


```

Original issue reported on code.google.com by `ejonesss...@gmail.com` on 14 Dec 2011 at 1:00

**Comment by GoogleCodeExporter** [comment]
```
Yes you do need a jailbreak to install iTransmission. Currently there is no 
jailbreak for iOS 4.3.5 on iPad 2. You can, however, update your iPad safely to 
iOS 5.0.1 and wait patiently for a jailbreak. iOS hacker pod2g is busy right 
now creating the untethered jailbreak for A4 devices (iPhone 4S and iPad 2 are 
A5) and as soon as he's done with those, he'll start on a jailbreak for A5 
devices. He recently said that it would be safe to update to 5.0.1 but to STAY 
AWAY FROM 5.1 WHEN IT COMES OUT. Hope this helped :)
```

Original comment by `kasim.ah...@gmail.com` on 18 Dec 2011 at 3:10

**Comment by GoogleCodeExporter** [comment]
```
if i put down the $99 for the developer kit and register my device is it the 
same as jailbreak as far as being able to run unsigned apps like yours?

also i saw a youtube video where you can get the free xcode tools and use that 
by swapping some payload packages from an unsigned package into a signed and 
some remapping it can do it without jailbreak

have you tried it and does it work.

```

Original comment by `ejonesss...@gmail.com` on 19 Dec 2011 at 4:05

**Comment by GoogleCodeExporter** [comment]
```
I DO NOT recommend you pay to be a developer unless you actually want to 
develop apps, which from what I hear, is extremely difficult and tedious. I 
would wait for a jailbreak. But if you can get the Xcode tools, then you MIGHT 
be able to install iTransmission without a jailbreak. And there is a way that 
you can install unsigned apps without a jailbreak or any Xcode tools. But al 
that does is install and let you run the app. Since iTransmission is only a 
torrent client and not a file manager, it will be come worthless. Until 
iTransmission becomes an actual file manager, you'll need a jailbreak to run it 
completely. Just wait. It shouldnt be long now :)
```

Original comment by `theinfec...@gmail.com` on 19 Dec 2011 at 2:07

**Comment by GoogleCodeExporter** [comment]
```
"extremely difficult and tedious"  what do you mean?

registering the device with apple?  where you have to go through a clunky 
process of restoring and updating to get itunes to pick up or have to remove 
all traces of itunes and quicktime in order to install the beta version of 
itunes?

why doesnt apple build in an uninstaller or something into the application.

also fortunately when i want to try something i have a usb drive i have 
installed osx on in order to isolate from the main system.

in a future version i dont know if apple can catch onto the bait and switch or 
even jumanji effect by writing an innocent download manager that gets onto the 
store and then with a very hidden secret code (like click on a certain icon 
while holding the home and power buttons while clicking the about or some part 
of the display and then enter some code to invoke an updater that installs the 
torrent part.

also what apple is doing  fighting the jailbreaks is illegal because not too 
long ago it was ruled that jailbreaking is legal
```

Original comment by `ejonesss...@gmail.com` on 19 Dec 2011 at 4:17

**Comment by GoogleCodeExporter** [comment]
```
tl;dr lol

The point is, making apps is difficult. Recently, iOS hacker pod2g cracked the 
iPhone 4S and a jailbreak is well on its way. I say sometime around next 
Thursday.
```

Original comment by `kasim.ah...@gmail.com` on 21 Dec 2011 at 1:51

**Comment by GoogleCodeExporter** [comment]
```
As for the legality of fighting jailbreaking:

Apple sometimes leaves security holes in iOS on accident which the jailbreak 
devs use. Apple then patches those to make the OS more secure. And jailbreaking 
being legal only ALLOWS us to jailbreak. If it REQUIRED us to jailbreak, then 
Apple would be in trouble.
```

Original comment by `kasim.ah...@gmail.com` on 21 Dec 2011 at 1:55

**Comment by GoogleCodeExporter** [comment]
```
you to think about it since i saw the youtube video that you can sign your own 
apps couldnt you sign or remove the need for signing and make jailbreaking not 
required.
```

Original comment by `ejonesss...@gmail.com` on 22 Dec 2011 at 3:10

**Comment by GoogleCodeExporter** [comment]
```
Yes but to me that's such a complicated process. From what I've heard, it works 
but I personally wait for the actual jailbreaks. This is just based on what you 
are able to do and what you want to do.
```

Original comment by `kasim.ah...@gmail.com` on 22 Dec 2011 at 3:31

**Comment by GoogleCodeExporter** [comment]
```
since apple does not allow torrent only clients couldnt you add ftp and other 
protocols support so apple would allow it on the store?
```

Original comment by `ejonesss...@gmail.com` on 22 Dec 2011 at 9:54

**Comment by GoogleCodeExporter** [comment]
```
I have no idea why you would need torrents or FTP to install an unsigned app on 
an iPad.
```

Original comment by `kasim.ah...@gmail.com` on 23 Dec 2011 at 1:13

**Comment by GoogleCodeExporter** [comment]
```
you dont but apple will not sign torrent only programs like yours so if you add 
an ftp or other clients to your program then apple will sign it allowing  
installing without jailbreaking, xcode file swap trickery as per the youtube 
instructions or $99 dev registration 

i am downloading the xcode tools right now (free at app store)
```

Original comment by `ejonesss...@gmail.com` on 26 Dec 2011 at 5:11

**Comment by GoogleCodeExporter** [comment]
```
i got it to install.

i found a site that sells developer certificate registrations for less and was 
able to sign the app and got it to install without jailbreak
```

Original comment by `ejonesss...@gmail.com` on 28 Dec 2011 at 7:04
